lib/sign_in/idme/service.rb

```diff
@@ -38,7 +39,9 @@ def normalized_attributes(user_info, credential_level)
                      when Constants::Auth::MHV
                        mhv_attributes(user_info)
                      end
-        attributes.merge(standard_attributes(user_info, credential_level))
+        attributes.merge(standard_attributes(user_info, credential_level)).tap do |attrs|
```

Comment:
I also wonder here if we should explicitly specify the fields that go in the digest, would be nice to include only user attribute data so we don't trigger a huge refresh of the digest when the ACR value changes